### PR TITLE
[DOCFIX] Add doc for Python client.

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: docs/en/Clients-Python.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Alluxio has a [Python Client](https://github.com/Alluxio/alluxio-py) for interacting with Alluxio through its
+[REST API](Clients-Rest.html). The Python client exposes an API similar to the [native Java API](Clients-Java-Native.html).
+See the [doc](http://alluxio-py.readthedocs.io) for detailed documentation about all available
+methods. See the [example](https://github.com/Alluxio/alluxio-py/blob/master/example.py) of how to perform basic filesystem
+operations in Alluxio.
+
+# Alluxio Proxy dependency
+
+The Python client talks to Alluxio through the REST API provided by the Alluxio proxy.
```
Comment (gpang) [inline docs/en/Clients-Python.md]: NIT: "talks to Alluxio" -> "interacts with Alluxio"

File: docs/en/Clients-Python.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+The proxy is a standalone server that can be started using
+`${ALLUXIO_HOME}/bin/alluxio-start.sh proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio-stop.sh
+proxy`. By default, the REST API is available on port 39999.
+
+There are performance implications of using the HTTP proxy. In particular, using the proxy requires
+an extra hop. For optimal performance, it is recommended to run the proxy server and an Alluxio
```
Comment (gpang) [inline docs/en/Clients-Python.md]: Will the "optimal" performance ever be as performant as co-located java client?

File: docs/en/Clients-Python.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+worker on each compute node.
+
+# Install Python Client Library
+```bash
+$ pip install alluxio
+```
+
+# Example Usage
+
+```python
```
Comment (gpang) [inline docs/en/Clients-Python.md]: Before this, include a short description of the setup and what this program does.

File: docs/en/Clients-Python.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+def info(s):
+    print green(s)
+
+
+def pretty_json(obj):
+    return json.dumps(obj, indent=2)
+
+
+py_test_root_dir = '/py-test-dir'
```
Comment (gpang) [inline docs/en/Clients-Python.md]: Should all this code be in a `main`?

File: docs/en/Clients-Python.md (hunk 1 above)
Comment (uronce-cc) [reply docs/en/Clients-Python.md]: done

File: docs/en/Clients-Python.md (hunk 2 above)
Comment (uronce-cc) [reply docs/en/Clients-Python.md]: no, it's as performant as the Go client.

File: docs/en/Clients-Python.md (hunk 3 above)
Comment (uronce-cc) [reply docs/en/Clients-Python.md]: done

File: docs/en/Clients-Python.md (hunk 4 above)
Comment (uronce-cc) [reply docs/en/Clients-Python.md]: done

**Comment by uronce-cc** [comment]
@gpang PTAL, thanks

**Pull request merged**